### DISCUSSION TITLE
Code security insights on the organization-level Security tab (Beta)

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#74687] [Code Security]
### Select Topic Area

Show & Tell

### Body

![image](https://github.com/community/community/assets/31521880/0253e783-1c15-47f0-b4a5-e6fdb9803b7f)
A new asset in security management is now available for GitHub enterprise users. Reinforcing the “shift left” philosophy, this feature is designed to integrate security into the heart of the development lifecycle, empowering your organization to proactively identify and address vulnerabilities.

# Key advantages
## Historical context
By comparing historical and current data, you can visibly track improvements in your security landscape and demonstrate the value of security investments.

![image](https://github.com/community/community/assets/31521880/85a16360-23b7-4d98-bf60-5662f126df4c)

## Customized focus
Sharpen your focus with filters that dissect your security data by teams, repositories, or any categorization that aligns with your goals. Whether it’s tracking team performance or monitoring metrics across a core group of repositories with the repository topic filter, there’s a plethora of options available to meet your needs.

![image](https://github.com/community/community/assets/31521880/0b7216ca-559f-40e8-bf2f-528de7ed6b0e)

## Prioritization made simple
With clear insights into severity and net resolve rate—security’s version of developer velocity—the dashboard shows you if your resources are aligned with the most severe threats and if remediation speed is in harmony with security demands.

![image](https://github.com/community/community/assets/31521880/a44cd098-246d-4565-b0a3-fb65987143b4)

## Strategic alignment
Gain a strategic perspective with the Repositories “Top 10” list, which shows you repositories with the largest number of open alert counts, to understand where to direct your attention first.

![image](https://github.com/community/community/assets/31521880/8966ed7e-9f3d-4bc0-bbd6-71bff5319047)

## Shift left
The dashboard, which is accessible by everyone in the organization, helps you drive best security practices by understanding potential issues as early as possible, reducing risk and workload down the line.

![image](https://github.com/community/community/assets/31521880/dfd7b457-f0a5-4210-9a7a-bc442f1303be)

This overview dashboard is now available as a beta on GitHub Enterprise Cloud and will be available in GitHub Enterprise Server 3.13.

[Learn more about the new overview dashboard](https://docs.github.com/en/enterprise-cloud@latest/code-security/security-overview/viewing-security-insights-for-your-organization) and [send us your feedback](https://github.com/orgs/community/discussions/categories/code-security)

**Comment by crs-k** [comment]
This feature is great. It would be helpful to be able to view the Security Coverage Trends metrics as a percentage of total repositories in scope. For organizations with hundreds of repositories, it would highlight coverage over time more effectively.

**Reply by kellyarwine** [reply]
Hi, @crs-k . Firstly, a big thank you for sharing your thoughts with us. As the product manager overseeing this area, I truly value the feedback we receive from our community.

I've gone through your comment a couple of times to ensure I fully grasp your perspective. However, I'd love to get a bit more clarity on your specific request. Could you please explain a bit more on what you're looking for?

**Reply by crs-k** [reply]
Below is an example of one of the charts showing dependabot alert coverage. If the total scope of my user's permission is two repositories like in the screenshot, it would be helpful to view this as a percentage (2 covered repos/2 total repos) of total repos for each data point. 

Over time, as coverage is added/removed and repositories are added/deleted, we can still set goals and track progress using the `percentage of coverage` for each category.

Notice at the top that there are 271 total repos and 259 users, so tracking `count of coverage` instead of `percentage of coverage` can be tough to value with a number of shifting variables. 

![image](https://github.com/community/community/assets/26232872/2552b294-db04-4e0c-8206-765b2b2dd359)

**Reply by kellyarwine** [reply]
Thank you, @crs-k! Understood. We'll work on it.

cc: @tassia-sp

**Comment by Yaf532** [comment]
[]()

**Comment by Yaf532** [comment]
[]()

**Reply by aaron-goshine** [reply]
When I export CSV I would like to see the serverity breakdown, not just a overall score .. I want to see how many criticals, highs ... similar to what is show on the risk tab page ...

**Comment by magdimikhail155** [comment]
Ikfdkhbf.  Fljnxx

**Comment by magdimikhail155** [comment]
Dfkybzxmnd.  Clibzh de

**Comment by matthewbush55** [comment]
When filtering by "tool" and "repo" and using "codeql" as the tool filter, I see a number of open alerts on the "Overview" page. When comparing this count to the count of open alerts in the repository view under "code scanning" and counting the number of unique alerts (only counting any duplicate alerts once), that number does not match the number of open alerts displayed in the Overview. Where is the number of open alerts on the "Overview" page derived from?

**Comment by jessehouwing** [comment]
Even though it's an unrelated feature, I'd love to see 2FA and SSO stats for orgs that aren't enforcing yet. We have a couple of Github orgs that are in the process of shoring up security and it takes time to reach all people. Having clear trends on number of users in the org, how many have 2fa, how many have sso would be useful information.

**Comment by denisstrelnikov** [comment]
I love these dashboards! One small feature request: could you add a filter by ﻿Configurations ([link](https://docs.github.com/en/code-security/securing-your-organization/enabling-security-features-in-your-organization/applying-the-github-recommended-security-configuration-in-your-organization))? We recently started using this feature and it would be perfect for filtering alerts on the dashboard. We’ve activated the configuration profile for our most critical repositories, and we want to prioritize tracking vulnerabilities in these repositories.

**Comment by zinping** [comment]
I love these dashboards! One small feature request: could you add a filter by ﻿Configurations ([link](https://docs.github.com/en/code-security/securing-your-organization/enabling-security-features-in-your-organization/applying-the-github-recommended-security-configuration-in-your-organization))? We recently started using this feature and it would be perfect for filtering alerts on the dashboard. We’ve activated the configuration profile for our most critical repositories, and we want to prioritize tracking vulnerabilities in these repositories.
